devel/ci-run-e2e.sh

```diff
@@ -40,3 +40,6 @@ echo "Running e2e test suite..."
 FLAKE_ATTEMPTS=2 "${SCRIPT_ROOT}/run-e2e.sh" \
   --ginkgo.skip=Venafi \
   "$@"
+
+echo "Exporting cluster logs to artifacts..."
+"${SCRIPT_ROOT}/cluster/export-logs.sh"
```

Comment:
This command won't be reached if the test fails because of `set -e`

Reply:
Whoops

Reply:
Fixed